adal/build.gradle

```diff
@@ -67,6 +67,7 @@ android {
         disable 'GradleDependency'
         disable 'GradleOverrides'
         disable 'OldTargetApi'
+        disable 'PackageManagerGetSignatures'
```

Comment:
This addition is unclear to me. @heidijinxujia What's the context for this one? Or the warning it triggers?